packages/preview/tonguetoquill-usaf-memo/0.1.0/README.md

```diff
@@ -0,0 +1,135 @@
+
+
+# tonguetoquill: USAF Memo Template for Typst
+
+
+[![github-repository](https://img.shields.io/badge/GitHub-Repository-blue?logo=github)](https://github.com/SnpM/tonguetoquill-usaf-memo)
+[![Nibs](https://img.shields.io/badge/author-Nibs-white?logo=github)](https://github.com/SnpM)
+
+[![typst-universe](https://img.shields.io/badge/Typst-Universe-aqua)](
+https://github.com/snpm/tonguetoquill-usaf-memo)
+
+A comprehensive Typst template for creating official United States Air Force memorandums that comply with AFH 33-337 "The Tongue and Quill" formatting standards.
+
+## Features
+
+- **Automatic formatting compliance** with AFH 33-337 standards
+- **Hierarchical paragraph numbering** (1., a., (1), (a)) with proper indentation
+- **Smart page break handling** for backmatter sections with continuation formatting
+- **Professional typography** with Times New Roman font and proper spacing
+- **Complete letterhead automation** including DoD seal placement and scaling
+- **Flexible content management** for various memo types
+- **Proper signature block positioning** and closing elements
+- **Intelligent space management** prevents orphaned headers and improper splits
+
+## Quick Start
+
+### Typst.app (Easiest)
+
+1. Go to [the package page](https://typst.app/universe/package/tonguetoquill-usaf-memo) and click "Create project in app".
+
+3. Download the [*CopperplateCC-Heavy*](https://github.com/SnpM/tonguetoquill-usaf-memo/blob/bebba4c1a51f9d67ca66e08109439b2c637e1015/template/assets/fonts/CopperplateCC-Heavy.otf) font and upload it to your project folder. This is an open-source clone of *Copperplate Gothic Bold*.
+    - **Note:** *Times New Roman* is a proprietary Microsoft font that I can't distribute legally. The package will automatically use the built-in *TeX Gyre Termes* font, an open-source clone of *Times New Roman*.
```

Comment:
The `Tex Gyre Termes` font isn't built-in from what I can tell, I had to download it manually.